client/src/components/common/common.js

```diff
@@ -1,17 +1,18 @@
 export { NavLink } from 'react-router-dom';
-export { default as Post } from './post/post.jsx';
-export { default as Icon } from './icon/icon.jsx';
-export { default as Image } from './image/image.jsx';
-export { default as Modal } from './modal/modal.jsx';
-export { default as Input } from './input/input.jsx';
-export { default as Header } from './header/header.jsx';
-export { default as Button } from './button/button.jsx';
-export { default as Segment } from './segment/segment.jsx';
-export { default as Message } from './message/message.jsx';
-export { default as Spinner } from './spinner/spinner.jsx';
-export { default as Checkbox } from './checkbox/checkbox.jsx';
-export { default as IconButton } from './icon-button/icon-button.jsx';
-export { default as PublicRoute } from './public-route/public-route.jsx';
-export { default as PrivateRoute } from './private-route/private-route.jsx';
-export { default as Notifications } from './notifications/notifications.jsx';
-export { default as CopyBufferInput } from './copy-buffer-input/copy-buffer-input.jsx';
+export * from './portal/portal.jsx';
```

Comment:
```suggestion
export { Portal } from './portal/portal.jsx';
```
export in this way

Reply:
@what1s1ove This approach related to only jsx components?

Reply:
@v0ldemar01 not, actually we want to remove "star" export/export

Reply:
I see, but as for this image, stay as prev?
![image](https://user-images.githubusercontent.com/43109766/168650586-9410ef69-c376-4f68-abbb-0c32344b7e3e.png)

Reply:
with the old one we will do it in the separate ticket